libraries/HTTPUpdate/src/HTTPUpdate.h

```diff
@@ -121,6 +126,7 @@ class HTTPUpdate
 private:
     int _httpClientTimeout;
     followRedirects_t _followRedirects;
+    String _md5Sum;
```

Comment:
`_md5Sum = String();` should be added in the class constructor after `_followRedirects` is set.

Reply:
But `String _md5Sum;` means it is automatically initialized empty String instance, isn't it?

Reply:
the compiler will complain. The String is not necessarily initialized. You can initialize it in two general ways:
```cpp
HTTPUpdate::HTTPUpdate(int httpClientTimeout): _httpClientTimeout(httpClientTimeout), _ledPin(-1), _md5Sum(){
  _followRedirects = HTTPC_DISABLE_FOLLOW_REDIRECTS;
}
```

Or
```cpp
HTTPUpdate::HTTPUpdate(int httpClientTimeout): _httpClientTimeout(httpClientTimeout), _ledPin(-1){
  _followRedirects = HTTPC_DISABLE_FOLLOW_REDIRECTS;
  _md5Sum = String();
}
```

Reply:
Ok, understood. Thanks for the correction 👍. Fixed.